### ISSUE TITLE
Correct bag-info.txt

### ISSUE THREAD

**Issue by dtarb** [hydroshare/hydroshare#154]
resourcemap.xml (Issues about resource maps moved to Issue #169 by Jeff)
- refers to resourcemetadata.json should point to XML
- resource slugs need to be corrected to GUIDs
- resolve domain's hydroshare.org that do not point to anything

bag-info.txt
- fix incorrect slug.  Should be gUID

**Comment by horsburgh** [comment]
Our packaging specification document states that we will not be using the following optional files from the BagIt specification:

bag-info.txt
tagmanifest-md5.txt

However, if these will be useful, they have already been implemented and we could keep them.  Right now, some of the information in bag-info.txt is semantically incorrect.  For example, my HydroShare user name and email is listed as the "author" for a resource I created, but I suspect that is owner information.  Not sure why we need to be putting this in an optional tag file anyway.  It just creates one more place where we have to keep it up to date.
